### ISSUE TITLE
Esbuild Error

### ISSUE THREAD

**Issue by mojtaba42** [cisco-open/otel-js#158]
I get this error when running a script bundled with esbuild:

I'm using the same code example for typescript in https://docs.telescope.app/javascript but it's embedded with an application bundled with esbuild

This is the error:

```
node:internal/validators:116
    throw new ERR_INVALID_ARG_TYPE(name, 'string', value);
    ^

TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined
    at new NodeError (node:internal/errors:387:5)
    at validateString (node:internal/validators:116:11)
    at Object.dirname (node:path:1276:5)
    at pkginfo.find (/Users/username/Downloads/dist_4/file.js:63194:20)
    at pkginfo.read (/Users/username/Downloads/dist_4/file.js:63219:26)
    at module2.exports (/Users/username/Downloads/dist_4/file.js:63176:25)
    at functions/filename/node_modules/@cisco-telescope/cisco-sdk-node/lib/src/tracing.js (/Users/username/Downloads/dist_4/filename.js:63289:22)
    at __require (/Users/username/Downloads/dist_4/filename.js:9:51)
    at functions/filename/node_modules/@cisco-telescope/cisco-sdk-node/lib/src/index.js (/Users/username/Downloads/dist_4/filename.js:63304:29)
    at __require (/Users/username/Downloads/dist_4/filename.js:9:51) {
  code: 'ERR_INVALID_ARG_TYPE'
}
```

As you can see it's related to this line:  https://github.com/cisco-open/otel-js/blob/main/packages/cisco-sdk-node/src/tracing.ts#L93

Other packages had similar issues with `pkginfo` like here https://github.com/slackapi/node-slack-sdk/issues/466#issuecomment-364729736

Is it possible to remove the dependency on `pkginfo` or figure out if we need specific context for bundling?

**Comment by osherv** [comment]
Hey, @mojtaba42 thanks for letting us know! Fixed, please update to the latest version (2.6.1)

**Comment by mojtaba42** [comment]
Thank you
